doc/gettingstarted.md

```diff
@@ -338,3 +342,9 @@ Now when the device sends an uplink, the application server will do a `POST` req
 	}
 }
 ``` 
+
+Use the `--help` flag to see all available options.
```

Comment:
If we mention `--help` here, I guess it could be removed from the gateway creation section.